api/routes/tasks.route.js

```diff
@@ -3,14 +3,16 @@ const router = express.Router();
 const taskValidator = require("../middlewares/validators/task.validator");
 
 const {
-  getAllTasks,
+  getUserTasks,
```

Comment:
Maybe tetAllTasks was better

Reply:
At the moment we are adding user to the app, may be it is better to call like this.